api/src/main/java/org/openmrs/module/pihcore/setup/LocationTagSetup.java

```diff
@@ -196,7 +196,7 @@ private static void setupLocationTagsForLiberia(LocationService locationService)
                 Arrays.asList(LiberiaLocations.OUTPATIENT_CLINIC, LiberiaLocations.MENTAL_HEALTH_CLINIC, LiberiaLocations.NCD_CLINIC, LiberiaLocations.MCH_CLINIC));
         // Mental health clinic dispense medications at the clinic - TODO - include pharmacies
         setLocationTagsFor(locationService, LocationTags.DISPENSING_LOCATION, Arrays.asList(LiberiaLocations.MENTAL_HEALTH_CLINIC));
-        setLocationTagsFor(locationService, LocationTags.MEDICAL_RECORD_LOCATION, Arrays.asList(LiberiaLocations.RECORDS_ROOM));
+        setLocationTagsFor(locationService, LocationTags.MEDICAL_RECORD_LOCATION, Arrays.asList(LiberiaLocations.RECORDS_ROOM, LiberiaLocations.EMERGENCY)); //this seem to be the main location for emergency registration
```

Comment:
I'm still not 100% sure what we want here, so let's not  just not add this for now, I don't think we need it.